R/github-labels.R

```diff
@@ -133,17 +133,18 @@ use_github_labels <- function(
     new_labels <- split(df$labels, df$number)
     purrr::iwalk(
       new_labels,
-      ~ gh(
-        "PATCH /repos/{owner}/{repo}/issues/{issue_number}",
-        issue_number = .y,
-        labels = I(.x)
-      )
+      \(x, y)
+        gh(
+          "PATCH /repos/{owner}/{repo}/issues/{issue_number}",
+          issue_number = y,
+          labels = I(x)
+        )
```

Comment:
Hmmm

Reply:
☝️ comment was just to facilitate some discussion of air formatting in a side channel.